menu/desktop-directories/lxqt-menu-system.directory.in

```diff
@@ -100,8 +100,7 @@ Comment[ca]=Preferències personals i dels programes d'administració
 Comment[cs]=Osobní volby a nastavení správy
 Comment[cy]=Hoffterau personol a gosodiadau gweinyddol
 Comment[da]=Personlige indstillinger og administrationskonfiguration
-Comment[de]=Persönliche Anpassungen und Administratoreinstellungen
-Comment[dz]=རང་དོན་དགའ་གདམ་དང་ བདག་སྐྱོང་གི་གཞི་སྒྲིག་ཚུ།
```

Comment:
dz translation accidentally dropped? That string is still in https://github.com/lxde/lxmenu-data/blob/991f14ba66f021fea8eaad023da10a04acf0b4a3/po/dz.po#L93

Reply:
I thought I had fixed that mistake...